src/test/functional/cypress/integration/feature/admin/empStatus.spec.js

```diff
@@ -102,7 +102,10 @@ describe('Admin - Employment Status', function () {
     it('update emp status', function () {
       cy.task('db:restore', {name: 'empStatus'});
       cy.loginTo(this.user, '/admin/employmentStatus');
-      cy.get(':nth-child(2) > .oxd-icon').click();
+      cy.wait('@getEmpStatus');
+      cy.get(
+        '.oxd-table-body > :nth-child(1) .oxd-table-cell-actions > :nth-child(2)',
```

Comment:
`.oxd-table-body > :nth-child(1)` - First row of cardtable
`.oxd-table-cell-actions > :nth-child(2)` - 2nd action of card table (edit)